regolith/builders/grantreportbuilder.py

```diff
@@ -54,14 +54,21 @@ def latex(self):
         begin_year, begin_month, begin_day = int(today[0])-1, int(today[1]), int(today[2])
         end_date = date(end_year, end_month, end_day)
         begin_date = date(begin_year, begin_month, begin_day)
+
+        # Get All Active Members
+        people = []
+        for p in self.gtx["people"]:
```

Comment:
``current_members = [person for person in self.gtx['people'] if person['active']]``